translations/binaries/slps_strings_city_names.json

```
@@ -0,0 +1,816 @@
+            "string": "セインガルドの西部にある村。[LINE]\n質素な村には不釣合いな大きな屋敷がある。[END]",
+            "original_bytes_length": 70,
+            "available_bytes_size": 71,
+            "translation": "세인갈드의 서부에 위치한 마을.[LINE]\n소박한 마을에는 어울리지 않는 큰 저택이 있다.[END]"
+        },
+        {
+            "offset": 1732464,
+            "string": "Harmentz[END]",
+            "translation": "Harmentz[END]",
+            "original_bytes_length": 9,
+            "available_bytes_size": 15
+        },
+        {
+            "offset": 1732480,
+            "string": "ハーメンツの村[END]",
+            "original_bytes_length": 15,
+            "available_bytes_size": 15,
+            "translation": "하멘츠의 마을[END]"
+        },
+        {
+            "offset": 1732496,
+            "string": "かつて勇名を馳せた船。[LINE]\n現在は海中深くに沈み、[LINE]\nその存在を知る者はいない。[END]",
+            "original_bytes_length": 73,
+            "available_bytes_size": 79,
+            "translation": "한때 용명으로 이름을 날렸던 배.[LINE]\n현재는 바다 깊숙이 가라앉아,[LINE]\n그 존재를 아는 자는 없다.[END]"
+        },
+        {
+            "offset": 1732576,
+            "string": "Radislowe[END]",
+            "translation": "Radislowe[END]",
+            "original_bytes_length": 10,
+            "available_bytes_size": 15
+        },
+        {
+            "offset": 1732592,
+            "string": "ラディスロウ[END]",
+            "original_bytes_length": 13,
+            "available_bytes_size": 15,
+            "translation": "라디슬로우[END]"
+        },
+        {
+            "offset": 1732608,
+            "string": "カルバレイスの西部にある街。[LINE]\n堆積した大昔の産業廃棄物の山を再利用して[LINE]\n生活する人々が住んでいる。[END]",
+            "original_bytes_length": 97,
+            "available_bytes_size": 103,
+            "translation": "칼바레이스의 서부에 위치한 도시.[LINE]\n퇴적된 아주 오래된 산업 폐기물의 산을 재활용하여[LINE]\n생활하는 사람들이 살고 있다.[END]"
+        },
+        {
+            "offset": 1732712,
+            "string": "Junkland[END]",
+            "translation": "Junkland[END]",
+            "original_bytes_length": 9,
+            "available_bytes_size": 15
+        },
+        {
+            "offset": 1732728,
+            "string": "ジャンクランド[END]",
+            "original_bytes_length": 15,
+            "available_bytes_size": 15,
+            "translation": "정크랜드[END]"
+        },
+        {
+            "offset": 1732744,
+            "string": "カルバレイスの首都。[LINE]\n店舗の軒先には、夜になると色とりどりの[LINE]\nネオンサインが輝く。[END]",
+            "original_bytes_length": 81,
+            "available_bytes_size": 87,
+            "translation": "칼바레이스의 수도.[LINE]\n상점의 처마에는 밤이 되면 다채로운[LINE]\n네온사인이 빛난다.[END]"
+        },
+        {
+            "offset": 1732832,
+            "string": "Kalviola[END]",
+            "translation": "Kalviola[END]",
+            "original_bytes_length": 9,
+            "available_bytes_size": 15
+        },
+        {
+            "offset": 1732848,
+            "string": "首都カルビオラ[END]",
+            "original_bytes_length": 15,
+            "available_bytes_size": 15,
+            "translation": "수도 칼비올라[END]"
```

```suggestion
            "translation": "수도 까르비올라[END]"
```

데스티니2에 "성지 까르비올라"라는 지명이 등장해서 일단 동일 명칭으로 해뒀어요-